executor/runtime/types/container.go

```diff
@@ -1318,3 +1330,25 @@ func isEFSID(FsID string) (bool, error) {
 	}
 	return matched, nil
 }
+
+// GetHumanFriendlyNetworkMode uses the incoming network mode string
+// and mutates it a bit to be a environment-variable safe string.
+// In the unknown mode, however, we return an empty string for
+// the caller to *not* set the variable
+func GetHumanFriendlyNetworkMode(mode string) string {
```

Comment:
In the proto object, there are two maps that have this info, I think?
```

var NetworkConfiguration_NetworkMode_name = map[int32]string{
        0: "UnknownNetworkMode",
        1: "Ipv4Only",
        2: "Ipv6AndIpv4",
        3: "Ipv6AndIpv4Fallback",
        4: "Ipv6Only",
}

var NetworkConfiguration_NetworkMode_value = map[string]int32{
        "UnknownNetworkMode":  0,
        "Ipv4Only":            1,
        "Ipv6AndIpv4":         2,
        "Ipv6AndIpv4Fallback": 3,
        "Ipv6Only":            4,
}

```

Why not use those names?

Reply:
I don't know if we want those internal names to be exposed to the user, maybe.

Here is the doc with the spec I'm conforming to:
https://docs.google.com/document/d/1IJxY4LaiVYy5RqXDRtYSscvysae4bB7-3v4SSqvdNbE/edit

Reply:
Hmm. Like with another enum?

Reply:
or a map of nice-names to less-nice-names (and vice-versa)?